content/cloud/cloud.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Cloud Computing'
+Description: 'Cloud Computing refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cloud-computing'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Cloud Computing** refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user. This is becoming more popular with businesses, and has even become ubiquitous in personal computing. Whenever a person sets up a website on GoDaddy, plays _World of Warcraft_, or streams a movie on Netflix, they are using a cloud computing solution.
+
+## Advantages
+
+Cloud computing offers the following advantages to the user:
+
+- Scalability: Adding resources is quicker and easier in a cloud environment then in an on-prem data center which involves purchasing, installing and configuring new hardware.
```

Comment:
```suggestion
- Scalability: Adding resources is quicker and easier in a cloud environment then in an on-premises data center which involves purchasing, installing and configuring new hardware.
```

I believe it's better we spell out "on-premises". Also, should these advantages be in **bold**?